Formula/g/glab.rb

```diff
@@ -1,10 +1,11 @@
 class Glab < Formula
   desc "Open-source GitLab command-line tool"
   homepage "https://gitlab.com/gitlab-org/cli"
-  url "https://gitlab.com/gitlab-org/cli/-/archive/v1.55.0/cli-v1.55.0.tar.gz"
-  sha256 "21f58698b92035461e8e8ba9040429f4b5a0f6d528d8333834ef522a973384c8"
+  url "https://gitlab.com/gitlab-org/cli.git",
+    tag:      "v1.55.0",
+    revision: "a806d3d2ae8515ca43be48a83598c04bb71c9328"
```

Comment:
Any specific reason for git instead of tarball archive download?

If it is needed for build / version info, which Go embeds automatically, it can also be added using Go's ldflags.

Reply:
Tarballs are preferred, but not really a blocker if there is specific need for git. 👍